bin/update

```diff
@@ -28,18 +40,20 @@ chdir APP_ROOT do
   # If we need a "reset all the things" script, we should call it bin/reset.
   system! 'bundle update'
 
+  # load 'Rakefile'
+  load_rakefile
   puts "\n== Updating database =="
-  system! 'bin/rails db:migrate'
+  Rake::Task["db:migrate"].invoke
 
   puts "\n== Seeding database =="
-  system! "bin/rails db:seed GOOD_MIGRATIONS=skip"
+  Rake::Task["db:seed"].invoke
```

Comment:
I like the idea but I think the problem is that we're invoking rake too many times, I think we can combine rake tasks first.  I don't remember why `db:seed` was split from `db:migrate`, maybe it was Chris' issue, but those can be combined in a single process.

So, while I like the idea this PR is trying to do, I think ultimately, it would be better to combine rake task invocations into multiple tasks per system call.